CIScripts/Build/BuildFunctions.ps1

```diff
@@ -256,6 +260,21 @@ function Invoke-AgentBuild {
     $Job.PopStep()
 }
 
+function Copy-Dlls {
+    Param (
+       [Parameter(Mandatory = $true)] [string] $OutputPath,
+       [Parameter(Mandatory = $false)] [bool] $ReleaseMode = $false
+    )
+
+    $Job.Step("Copying dlls to $OutputPath", {
+        if (-not $ReleaseMode) {
```

Comment:
We're thinking that this `if` should be outside the `Copy-Dlls` function.

```
if(-not $ReleaseMode) {
 Copy-Dlls -OutputPath ......
}
function Copy-Dlls {
  params.
  job step
    cp x y 
}
```
Also, maybe we should rename the function to `Copy-VCDebugDlls`.

Reply:
If we're not going to copy any DLLs for release mode, then sure.